### PR TITLE
Standardize the formatting of local labels

### PR DESCRIPTION
**Pull request by kqesar**
Standardize the name of functions (begin with `.` like `.bagFull`) in pascalCase

### CODE REVIEW

File: engine/movie/evolution.asm [hunk 1]
```diff
@@ -97,24 +97,24 @@ EvolutionSetWholeScreenPalette:
 	ld b, SET_PAL_POKEMON_WHOLE_SCREEN
 	jp RunPaletteCommand
 
-Evolution_LoadPic:
+EvolutionLoadPic:
```
Comment (dannye) [inline engine/movie/evolution.asm:100]: I like the underscores that are used to separate the subject matter on the left from the specific action on the right.
This is [something that I continue to use in poketcg](https://github.com/pret/poketcg/blob/master/src/engine/challenge_machine.asm) for example.

File: scripts/BluesHouse.asm [hunk 2]
```diff
@@ -46,12 +46,12 @@ BluesHouseDaisySittingText:
 	SetEvent EVENT_GOT_TOWN_MAP
 	jr .done
 
-.got_town_map
+.gotTownMap
```
Comment (dannye) [inline scripts/BluesHouse.asm:49]: I'm definitely fine with changing `.Done` to `.done` etc, but I've gotten real used to the easier readability of purely lowercase `.snake_case` labels. I'm not sure I'm ready to see it go.

**Comment by dannye** [comment]
It looks like you mostly focused on the scripts/ folder, but there are several more `.PascalCase` labels in audio/, data/, engine/, and home/.

**Comment by KqesaR** [comment]
Yes for the moment all functions are in script folder.

I can continue on other directory if it's ok for the snake_case as case convention for functions

**Comment by KqesaR** [comment]
Ok @dannye , i keep the camelCase and use it for other folder :)

**Comment by Rangi42** [comment]
> Now, between `.camelCase` and `.snake_case`, my preference is for `.snake_case`, but given that it's only 10% of all local labels, I'd be okay with going all-in with `.camelCase` if that's what everyone else thinks.

I actually also prefer `.snake_case` for local labels. Whatever style we end up with, though, should be applied to all the repos (red, yellow, gold, and crystal). (yellow can probably merge the PR from red, but may also need some of its own labels updated, e.g. for Pikachu or GBC-specific code.)

> I think we are all in agreement about changing `.PascalCase` to `.camelCase`.

Generally yes, but I think some of those might be in `.PascalCase` because they're more like independent sub-sub-routines than just local pieces of one large subroutines. So on a case-by-case basis we may want to make them non-local `PascalCase` labels instead, even if they're only called once or twice by the subroutine above them.

**Comment by dannye** [comment]
> Generally yes, but I think some of those might be in `.PascalCase` because they're more like independent sub-sub-routines than just local pieces of one large subroutines. So on a case-by-case basis we may want to make them non-local `PascalCase` labels instead, even if they're only called once or twice by the subroutine above them.

That's fair. I was thinking a similar thing with some of the `.snake_case` local labels.
I think there are a few local labels with underscores that -- even if we switched to `.camelCase` -- should keep their underscore for situation-specific reasons. Kind of like `Evolution_LoadPic` discussed above, but for local labels.

**Comment by Rangi42** [comment]
I agree about `Evolution_LoadPic` and such.

**Comment by mid-kid** [comment]
Humble mention of https://github.com/pret/pokecrystal/blob/master/STYLE.md

**Pull request closed without merging**